guides/common/attributes-satellite.adoc

```diff
@@ -136,27 +136,28 @@
 
 // For a release
 :RepoSatelliteVersion: {ProjectVersion}
+:project-client-2-name: satellite-6-client-2
```

Comment:
I am unsure if it's a good idea to have the number as part of the attribute name.

Reply:
@maximiliankolb should I add the `2` to the repo labels?

Reply:
Hmm, I again wonder if my original comment was a good idea. Maybe your solution works best? We already use "project-client-name". I am OK as is as well as adding `-2` to all three attributes.